pruntime/enclave/src/contracts/balance.rs

```diff
@@ -21,6 +22,8 @@ pub struct Balance {
     accounts: BTreeMap<AccountIdWrapper, chain::Balance>,
     sequence: SequenceType,
     queue: Vec<TransferData>,
+    secret: Option<[u8; 32]>,
+    machine_id: Option<[u8; 16]>,
```

Comment:
Skip serialization for these fields because they are not part of the contract states